services/dashboard/app/root.tsx

```diff
@@ -81,6 +81,7 @@ export const meta: MetaFunction<typeof loader> = ({ data }) => {
 export async function loader({ request }: DataFunctionArgs) {
 	const timings = makeTimings('root loader')
 
+	const workflow = await fetch('https://localhost:7777/workflow').then(res => res.json())
```

Comment:
In the toolkit of this stack of technologies, there is a concept named [Server timing](https://github.com/epicweb-dev/epic-stack/blob/main/docs/server-timing.md) can you take a look and review if you can implement? If take more than 30 minutes, just let me, @DanielMoralesRosales or @veroPott know, and ask for a separate ticket for it

Reply:
I think I would need that separated ticket from it:(, I can't seem to make the server timings to show on the network tab